File: src/main/java/Casino.java
package main.java;

import java.io.File;

public class Casino {

    public static void main(String[] args) {
        Hand sam = new Hand("Sam");
        Hand dealer = new Hand("Dealer");

        if (args.length > 0) {
            File file = new File(args[0]);
            if (file.exists()) {
                new BlackJack(Deck.createDeckFromFile(file), dealer, sam);
            }

        } else {
            new BlackJack(Deck.getNewShuffledDeck(), dealer, sam);
        }
    }

}

File: src/main/kotlin/Casino.kt
package main.kotlin

import main.kotlin.Deck.createDeckFromFile
import main.kotlin.Deck.newShuffledDeck
import kotlin.jvm.JvmStatic
import java.io.File

object Casino {
    @JvmStatic
    fun main(args: Array<String>) {
        val sam = Hand("Sam")
        val dealer = Hand("Dealer")
        if (args.isNotEmpty()) {
            val file = File(args[0])
            if (file.exists()) {
                BlackJack(createDeckFromFile(file), dealer, sam)
            }
        } else {
            BlackJack(newShuffledDeck(), dealer, sam)
        }
    }
}